source/features/fix-no-pr-search.tsx

```diff
@@ -5,11 +5,22 @@ import features from '../feature-manager.js';
 import SearchQuery from '../github-helpers/search-query.js';
 
 function redirectToIssues(event: DelegateEvent<Event, HTMLFormElement>): void {
+	event.preventDefault();
```

Comment:
What happens if this is missing? Is GitHub going ahead and loading it via ajax?

Reply:
like this 👇.

https://github.com/user-attachments/assets/19b9086d-42b6-442d-bc97-9f745aa29673

Reply:
> Is GitHub going ahead and loading it via ajax?

The answer is yes. They ajaxed the form.

Reply:
If I don't add `submit()` the active tab will not change, it stay in pull requests

Reply:
<img width="800" alt="image" src="https://github.com/user-attachments/assets/c862e836-717b-40be-8801-969e7392bb60">

Reply:
Got it 🙏 thanks!